tests/integration/suite/test_kontainer_drivers.py

```diff
@@ -271,6 +272,28 @@ def test_update_duplicate_driver_conflict(admin_mc, wait_remove_resource):
         assert "Driver URL already in use:" in e.error.message
 
 
+def test_kontainer_driver_links(admin_mc):
+    client = admin_mc.client
+    lister = client.list_kontainerDriver()
+    assert 'rancher-images' in lister.links
+    token = 'Bearer '+client.token
+    data = requests.get(
+            url="https://localhost:8443/v3/kontainerdrivers/rancher-images",
```

Comment:
If there is no library support for collection link, and we are building the request ourselves, can we build the url similar to how library does it? hostname in particular

Reply:
Thank you, made me look at how we were generating base_url. Updated.